nvflare/app_common/utils/fl_model_utils.py

```diff
@@ -205,6 +213,9 @@ def update_model(model: FLModel, model_update: FLModel, replace_meta: bool = Tru
             model.meta = model_update.meta
```

Comment:
There's a typo in the RuntimeError above. The message should be `f"params_type {model.params_type} of `model`...`